cws-installer/src/main/java/jpl/cws/task/CwsInstaller.java

```diff
@@ -2020,6 +2060,28 @@ private static void updateFiles() throws IOException {
 		content = content.replace("__CWS_DB_DRIVER__",             cws_db_driver);
 		content = content.replace("__CWS_DB_USERNAME__",           cws_db_username);
 		content = content.replace("__CWS_DB_PASSWORD__",           cws_db_password);
+
+		writeToFile(filePath, content);
+		copy(
+			Paths.get(config_work_dir + SEP + "tomcat_conf" + SEP + "server_adaptation.xml"),
+			Paths.get(cws_tomcat_root + SEP + "conf"        + SEP + "server_adaptation.xml"));
+		if (cws_adaptation_db_url != null && cws_adaptation_db_driver != null &&
+			cws_adaptation_db_username != null && cws_adaptation_db_password != null) {
+
+			// Fill in the __ADAPTATION_SERVER_RESOURCE__
+			String serverAdaptationContent = getFileContents(
+				Paths.get(config_work_dir + SEP + "tomcat_conf" + SEP + "server_adaptation.xml"));
+			content = content.replace("<Resource name=\"jdbc/cws\"", serverAdaptationContent);
+
+			content = content.replace("__CWS_ADAPT_DB_URL__",          cws_adaptation_db_url);
+			content = content.replace("__CWS_ADAPT_DB_DRIVER__",       cws_adaptation_db_driver);
+			content = content.replace("__CWS_ADAPT_DB_USERNAME__",     cws_adaptation_db_username);
```

Comment:
Can you be consistent on your naming convention?  Make these __CWS_ADAPTATION_ also.

Reply:
I would also vote to be consistent, but maybe have both be the shorter "ADAPT" form?

Reply:
renamed the cws adapt variables to `cws_adapt_`